### PR TITLE
Add SimSim.app 1.3.3

### PR DESCRIPTION
**Pull request by rtdaly**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/simsim.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  url "https://github.com/dsmelov/simsim/releases/download/#{version}/SimSim_#{version}.dmg"
+  appcast 'https://github.com/dsmelov/simsim/releases.atom',
+          checkpoint: '0b8129ef11a6ef819d26b1485f4afd90444ba05189aa4bab86c199e605432571'
+  name 'SimSim'
+  homepage 'https://github.com/dsmelov/simsim/'
+
+  app 'SimSim.app'
+
+  uninstall login_item: 'SimSim',
```
Comment (commitay) [inline Casks/simsim.rb]: Does this need uninstall `quit:` or `signal:`?

File: Casks/simsim.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            kext:       'com.dsmelov.SimSim'
+
+  zap trash: [
+               '~/Library/Preferences/com.dsmelov.SimSim.plist',
```
Comment (commitay) [inline Casks/simsim.rb]: Make this a single line with `zap trash`.

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/simsim.rb (hunk 1 above)
Comment (rtdaly) [reply Casks/simsim.rb]: You're right I forgot the `quit:`

**Comment by rtdaly** [comment]
@commitay Fixed!

**Review by commitay — APPROVED** [review]

**Pull request merged**